app/models/article.rb

```diff
@@ -817,6 +817,7 @@ def notify_slack_channel_about_publication
   end
 
   def detect_animated_images
+    return unless FeatureFlag.enabled?(:detect_animated_images)
```

Comment:
Did we add the feature flag via a DUS previously? If not, do you want to add that to this PR, I know you generally advocate for this.

Reply:
Normally I would have but it was a bit of an emergency and I still don't know if this was only happening on DEV or elsewhere. Will formalize when we know more